### ISSUE TITLE
The specified deps.json [(projectpath).deps.json] does not exist

### ISSUE THREAD

**Issue by Zefek** [dotnet/efcore#24884]
Hello,

I have same problem as in issue 16386. I have latest version Visual Studio (16.9.4). But same issue I have on older version (16.9.2, 3...) too. Another machine works fine. 
Commands add-migration, dbcontext, update-database stops on error The specified deps.json [(projectpath).deps.json] does not exist. It is true,  deps.json file is not exists in this directory because it is root of project with many .cs files.

I tried to run command with -verbose option and then change path where deps.json file is - bin/debug/netcoreapp3.1. But there is error:
info:    System.IO.FileLoadException: Could not load file or assembly 'netstandard, Version=2.1.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)
info:    File name: 'netstandard, Version=2.1.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'
info:       at Microsoft.EntityFrameworkCore.Design.Internal.DbContextOperations.GetContextInfo(String contextType)
info:       at Microsoft.EntityFrameworkCore.Design.OperationExecutor.GetContextInfoImpl(String contextType)
info:       at Microsoft.EntityFrameworkCore.Design.OperationExecutor.OperationBase.<>c__DisplayClass3_0`1.<Execute>b__0()
info:       at Microsoft.EntityFrameworkCore.Design.OperationExecutor.OperationBase.Execute(Action action)
info:
info:
error:   Could not load file or assembly 'netstandard, Version=2.1.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)

What should be wrong and how can I solve this problem? 
Thank you.

**Comment by ajcvickers** [comment]
@Zefek This is an issue with the project system that we have not been able to reproduce. Ultimately it will be up to the project system team to fix this, but if you are able to figure out the steps to get into this state such that this can be reproduced, then we will pass that information on to the project team.

**Comment by Zefek** [comment]
I understand. But I have question. I compared verbose output with my colleague. Command from Package  Console Manager call this command: "C:\Program Files\dotnet\dotnet.exe" exec --depsfile ... This is first difference with ma colleague output. I have this parameter set to folder which is project root. My colleague has bin/debug/netcoreapp3.1 folder. How Package Console Manager determine this path? 

I tried command from this comment (https://github.com/dotnet/efcore/issues/16386#issuecomment-509090715) with same result. Item('OutputPath') is null. But I didn't found any information how to I can OutputPath set up. 

Thank you for answer.
